src/centralserver/admin-ui/src/views/ManagementRequests/ManagementRequests.vue

```diff
@@ -115,44 +115,39 @@ export default Vue.extend({
           created: '2021-02-01',
           type: 'change_owner',
           serverOwnerName: 'Tartu Kesklinna Perearstikeskus OÜ',
-          serverOnwerId: 'DEV-333',
-          serverCode: 'sidecar',
+          serverId: 'DEV-333:sidecar',
           status: 'APPROVED',
         },
         {
           id: '736287',
           created: '2021-05-05',
           type: 'delete_certificate',
           serverOwnerName: 'Eesti Põllumajandusloomade Jõudluskontrolli ASi',
-          serverOnwerId: 'COM-777',
-          serverCode: 'SS1',
+          serverId: 'COM-777:SS1',
```

Comment:
The mock data is not up to date the with opendapi definition (or current implementation). More accurate would be something like:
```
{
  "id": 13,
  "type": "AUTH_CERT_REGISTRATION_REQUEST",
  "origin": "CENTER",
  "server_owner_name": "...",
  "security_server_id": {
    "instance_id": "DEV7X",
    "type": "SERVER",
    "member_class": "TST",
    "membe_crode": "MEMBER1",
    "server_code": "RH1"
  },
  "status": "APPROVED",
  "created_at": "2021-07-07T10:09:42.10186Z"
}
```

Reply:
Mock data has been updated to look more like the real one